Source/WebKit/UIProcess/API/ios/WKWebViewIOS.h

```diff
@@ -182,6 +182,10 @@ enum class TapHandlingResult : uint8_t;
 @property (nonatomic, readonly) BOOL _isWindowResizingEnabled;
 #endif
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+@property (nonatomic, readonly) BOOL _isLinkPreview;
```

Comment:
Should this be in `WKWebViewPrivate.h`?

Reply:
Hmm, I wonder if this should be a bit that's about the thing you're disabling, not a bit about "is a link preview" that happens to implicitly disable it. I think that's how we usually do this. (I can imagine that there would be other uses for that, also...)

Reply:
I ended up not needing any additional state variables so I got rid of this